libs/ChatSection/include/Status/ChatSection/ChatSectionController.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "ChatItem.h"
+#include "ChatDataProvider.h"
+
+#include <Helpers/QObjectVectorModel.h>
+
+namespace Status::ChatSection {
+
+    class ChatSectionController: public QObject
+    {
+        Q_OBJECT
+        QML_ELEMENT
+
+        Q_PROPERTY(QAbstractListModel* chatsModel READ chatsModel NOTIFY chatsModelChanged)
+        Q_PROPERTY(ChatItem* currentChat READ currentChat NOTIFY currentChatChanged)
+
+    public:
+        ChatSectionController();
+
+        QAbstractListModel* chatsModel() const;
+        ChatItem* currentChat() const;
+
+        Q_INVOKABLE void init(const QString& sectionId);
```

Comment:
@stefandunca cause it's a different story here, when I was working on this I was thinking about communities as well where this ChatSection plugin will be shared between chat and community sections. Now since number of communities is not known in advance this is one of the possible ways to go in order to reuse the same plugin and set everything properly.